packages/opds-web-client/package.json

```diff
@@ -13,9 +12,9 @@
   "scripts": {
     "lint": "tslint -c tslint.json src/*.ts src/*.tsx src/**/*.ts src/**/*.tsx src/**/**/*.ts src/**/**/*.tsx && sass-lint -c .sass-lint.yml -v -q",
     "test": "npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/testHelper.js lib/__tests__/*.js lib/**/__tests__/*.js lib/**/**/__tests__/*.js",
+    "test-file": "npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/testHelper.js",
     "dev": "webpack-dev-server --progress --history-api-fallback --hot --content-base build --port 8090 --config webpack.dev.config",
-    "prod": "NODE_ENV=production webpack -p --progress --display-modules --config webpack.prod.config && rm -rf dist/lib",
-    "prepublish": "typings install && rm -rf lib && tsc && cp -r src/stylesheets lib && npm run prod"
```

Comment:
Why is this gone instead of moved to `prepublishOnly`?

Reply:
I'm still trying to figure this part out. But now I realized where I could've gone wrong. I just realized that the `prepublish` command builds the lib folder.